README.md

```diff
@@ -1,2 +1,5 @@
-# docs
-Luna documentation viewer
+
+# LunaDocs
+
+Documentation application for most sophisticated and futuristic programming language - Luna - a hybrid visual language with textual representation. Check it out at http://luna-docs.herokuapp.com/
```

Comment:
We should describe Luna using some less hyped words imo ;) People usually don't like project authors to describe projects in such way. It is very nice from your side Maciek though! :)